qcodes/instrument_drivers/QuTech/IVVI.py

```diff
@@ -183,7 +183,45 @@ def _get_version(self):
             mes = self.ask(bytes([3, 4]))
             ver = mes[2]
             return ver
-
+    
+    def linsweep(self,start,end,points):
+        """
+        A replacement of numpy.linspace for use with the ivvi rack dacs.
+        ivvi.linsweep gives:
+         - even spacing in voltages
+         - voltages match the set voltages
+         - protection from oversampling
+         
+        Args:
+            start (number): starting voltage
```

Comment:
no need for type information in the docstring thanks to python type annotations. (moreover, what is a "number"? "float or int"?)
```suggestion
            start: starting voltage
```